index.html.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
- You have rightly attempted to link your stylesheet here but you will notice that the style was not applied. Did you notice that too?
- See what it looks like :point_down: :

![image](https://user-images.githubusercontent.com/99715640/212387598-e53120ea-5257-49fc-89a6-bf5e5f063876.png)

- [ ] This is happening because of some naming issues between your files and your `html` document. Kindly check your files and you will discover that you made a mistake by creating a `style.css.css` file instead of the correct one you were trying to link - which is `style.css`. Please try to rename this file to match. Also, the same mistake was made with the `index.html.html` - kindly fix both please  :hammer_and_wrench: